.gitignore

```diff
@@ -7,6 +7,7 @@ runs/
 .cache/
 .eggs/
 lightning_logs/
+outputs/
```

Comment:
why is this change necessary?

Reply:
`hydra` adds this automatically as part of its experiment management tracker. Will need to look into how to disable it.